CITATION.cff

```diff
@@ -0,0 +1,59 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it using these metadata."
+title: "Vitessce"
+authors:
+  - given-names: "Mark S."
+    family-names: "Keller"
+    affiliation: "Harvard Medical School"
+    orcid: "https://orcid.org/0000-0003-3003-874X"
+  - given-names: "Ilan"
+    family-names: "Gold"
+    orcid: "https://orcid.org/0000-0002-5823-1026"
+    affiliation: "Harvard Medical School"
+  - given-names: "Trevor"
+    family-names: "Manz"
+    orcid: "https://orcid.org/0000-0001-7694-5164"
+    affiliation: "Harvard Medical School"
+  - given-names: "Iva"
+    family-names: "Babukova"
+    affiliation: "Biomage"
+  - given-names: "Chuck"
+    family-names: "McCallum"
+    affiliation: "Harvard Medical School"
+    orcid: "https://orcid.org/0000-0003-4039-9768"
+  - family-names: "Gehlenborg"
+    given-names: "Nils"
+    orcid: "https://orcid.org/0000-0003-0327-8297"
```

Comment:
no affiliation for Nils?

Reply:
Whoops, thanks for catching this, just updated